test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json

```diff
@@ -1388,7 +1388,7 @@
         "_score": 1.0,
         "_source": {
             "entity_id": "aaa96233-bf27-44c7-82df-b4dc15ad4d9d",
-            "source": [{
+            "sources": [{
```

Comment:
This looks like it is in the wrong commit, or needlessly split over two commits. The first commit calls this `source` the second one calls this `sources`. The intermediate `source` name shouldn't be visible in the history.

Reply:
:facepalm: nice catch.